mtp_api/apps/mtp_auth/admin.py

```diff
@@ -34,6 +37,17 @@ class RoleAdmin(ModelAdmin):
     def all_groups(self, instance):
         return ', '.join(sorted(group.name for group in instance.groups))
 
+@admin.register(JobInformation)
+class JobInformationAdmin(ModelAdmin):
+    list_display = ('title', 'prison_estate', 'tasks', 'fullname', 'email', 'user_id')
```

Comment:
i don't think `user_id` is useful to anybody, but i suppose you could have `username` instead

Reply:
Good point, we have updated that